packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -1,10 +1,10 @@
 import React, { forwardRef } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 import { AngleDownIcon, AngleUpIcon } from '@commercetools-uikit/icons';
 import IconButton from '@commercetools-uikit/icon-button';
-import SpacingsInline from '@commercetools-uikit/spacings-inline';
```

Comment:
Why did you need to switch to manual spacing?   Maybe you don't need to do that any more when always having a default value for context?

Reply:
I changed to a flexbox approach so that the title automatically rearrange on 2 lines in case the viewport is too narrow. In order to do that I had to get rid of the `SpacingsInline` which, if I don't understand wrong, is arranging all the elements in a single row and doesn't allow the content to "flow" on multiple rows.

> Maybe you don't need to do that any more when always having a default value for context?
Not sure I understand what you mean with default value for context